rust/.cargo/config.toml

```diff
@@ -11,4 +12,3 @@ rustflags = [
     "-Cllvm-args=-enable-emscripten-cxx-exceptions=0",
```

Comment:
I think we no longer need the lines I added in 6264445fdd714b38e3fbf8bcde1a19a54b696703, including this one, which is very similar to the line you added?